lib/ruby_llm/providers/bedrock/tools.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module RubyLLM
+  module Providers
+    module Bedrock
+      # Tools methods for the AWS Bedrock API implementation
+      module Tools
+        module_function
+
+        def tool_for(tool)
```

Comment:
Just realized I have not tested this.

Reply:
![image](https://github.com/user-attachments/assets/8eae1901-cef5-4360-9854-8d89f1e03fbc)
Looks like it works.

Reply:
Scratch that. Revisiting this.

Reply:
Got it to work.